utils/scapp.h

```diff
@@ -6,6 +6,8 @@
 
 #include <zstd.h>
 #include "ktxapp.h"
+#include "ktxint.h"
```

Comment:
On windows/VS 'max' isn't defined and instead the macro 'MAX' is used throughout the project which is defined in `ktxint.h`. Instead of redefining it in here I have included the header instead. 

Would you rather me do:

```
#ifndef MAX
#define MAX(x, y) (((x) > (y)) ? (x) : (y))
#endif
```

Here again?

Reply:
I'm not finding any use of MAX in scapp.h, ktxsc or toktx. Is this something you started using in this PR? I suggest instead using `std::max` as we require at least C++11 for these apps.

Reply:
> Is this something you started using in this PR?

Yes. In https://github.com/wasimabbas-arm/KTX-Software/blob/astc_support/utils/scapp.h#L455. 

> I suggest instead using std::max as we require at least C++11 
In my mind I was using `std::max` just [before](https://github.com/KhronosGroup/KTX-Software/pull/433/commits/aff59edcf922f1c6758012793beecd2c803447a7#diff-a28f7962aaf3829aaac3950ff70e3943c119d108eba8ee03d821f2ec64a32a37L372) the [fix](https://github.com/KhronosGroup/KTX-Software/pull/433/commits/b382950903622a1f557187f76457bbf15654f768) but I think what has happend is that other compilers were more forgiving than VS regarding leaving the `std::`. I thought I did check this but that might have been the case.

Fixed in https://github.com/KhronosGroup/KTX-Software/pull/433/commits/625b5eaf50f89f96f6790082aa314c2181510398 lets see what VS says.

Reply:
You need `#include <algorithm>`. That will likely stop VS complaining, if indeed it does.

Reply:
Ah yes. That would help. 2019 VS version is happy but 2015 and 2017 not. https://ci.appveyor.com/project/Khronoswebmaster/ktx-software/builds/40159472. Added `#include <algorithm>` now as well.